tests/test_ppo_trainer.py

```diff
@@ -11,1271 +11,53 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
This one is just the tests/test_ppov2_trainer.py renamed. The diff view makes no sense